galaxy/main/signals/handlers.py

```diff
@@ -64,18 +60,26 @@ def user_logged_in_handler(request, user, **kwargs):
         'email': None,
         'html_url': social.extra_data.get('blog'),
     }
-    if not namespace:
-        # Only create one Namespace
-        namespace, _ = models.Namespace.objects.get_or_create(
-            name=sanitized_username, defaults=defaults)
-        namespace.owners.add(user)
-    defaults['description'] = social.extra_data.get('bio') or name
-    defaults['followers'] = social.extra_data.get('followers')
-    defaults['display_name'] = name
-    defaults['avatar_url'] = social.extra_data.get('avatar_url')
+
+    ns_defaults = {'name': sanitized_username, **defaults}
+
+    # Create lowercase namespace if case insensitive get does not find match
+    namespace, _ = models.Namespace.objects.get_or_create(
```

Comment:
`.get` throws an error if two items match the search. If `.get_or_create` works the same way, will this cause an exception if there are two namespaces with different capitalization?

Reply:
I hadn't tested when there already was a duplicated namespace, I will check and fix if needed. Thanks!